.env.local.template

```diff
@@ -1 +1,2 @@
-MONGO_URI=""
\ No newline at end of file
+MONGO_URI=""
+SECRET=""
```

Comment:
@Sandeep1908 rename this to `JWT_SECRET` and also update the code wherever you're using this

Plus this env template file doesn't have `PORT` please add that too.